template/python-flake/flake.nix

```diff
@@ -0,0 +1,51 @@
+{
+  description = "Your jupyterWith project";
+
+  nixConfig.extra-substituters = [
+    "https://tweag-jupyter.cachix.org"
+  ];
+  nixConfig.extra-trusted-public-keys = [
+    "tweag-jupyter.cachix.org-1:UtNH4Zs6hVUFpFBTLaA4ejYavPo5EFFqgd7G7FxGW9g="
+  ];
+
+  inputs.flake-compat.url = "github:edolstra/flake-compat";
+  inputs.flake-compat.flake = false;
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+  # TODO: inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/1158501e7c7cba26d922723cf9f70099995eb755";
```

Comment:
Any reason why this is hardcoded?

If this is just to get a fixed known version, maybe we could ship a (partial) `flake.lock` that pins it (but doesn't prevent `nix flake update` from working)

Reply:
Or have `inputs.nixpkgs.follows = "jupyterWith/nixpkgs"` to make sure that both stay in sync if that's desirable